Doc/c-api/object.rst

```diff
@@ -575,3 +575,18 @@ Object Protocol
    has the :c:macro:`Py_TPFLAGS_MANAGED_DICT` flag set.
 
    .. versionadded:: 3.13
+
+.. c:function:: int PyUnstable_Object_SetDeferredRefcount(PyObject *obj)
```

Comment:
Let's rename this to `PyUnstable_Object_EnableDeferredRefcount` per Petr's suggestion.